applications/vessel-history/src/features/profile/components/activity/Activity.tsx

```diff
@@ -37,6 +39,10 @@ const Activity: React.FC<InfoProps> = (props): React.ReactElement => {
     dispatch(toggleGroup({index}))
   },[dispatch])
 
+  useEffect(() => {
+    dispatch(fetchRegionsThunk())
```

Comment:
If we're going to call `fetchRegionsThunk` every time the user enters a vessel profile, I'd add a condition to the thunk so that it performs the request to the API only once.